.github/workflows/docker.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          dockerfile: exe/server/Dockerfile
+          dockerfile: cmd/powd/Dockerfile
```

Comment:
Just a remaining side-effect of previous folder rename. (Not related with PR topic).